<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RulerDlg</class>
 <widget class="QDialog" name="RulerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>165</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>165</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>165</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ruler</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QTabWidget" name="rulerModeTab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>380</width>
     <height>121</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="lineTab">
    <attribute name="title">
     <string>Line</string>
    </attribute>
    <widget class="QLabel" name="lineLengthValue">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>20</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lineAngleValue">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>50</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="lineMetricsCombo">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>3</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
     <property name="maxVisibleItems">
      <number>20</number>
     </property>
    </widget>
    <widget class="QLabel" name="lineAngleUnit">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>degrees</string>
     </property>
    </widget>
    <widget class="QLabel" name="lineLengthTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>65</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>3</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
     <property name="text">
      <string>Length:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lineAngleTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>65</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Heading:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pathTab">
    <attribute name="title">
     <string>Path</string>
    </attribute>
    <widget class="QComboBox" name="pathMetricsCombo">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxVisibleItems">
      <number>20</number>
     </property>
    </widget>
    <widget class="QLabel" name="pathLengthTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>65</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Length:</string>
     </property>
    </widget>
    <widget class="QLabel" name="pathLengthValue">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>20</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>135</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="mouseCheck">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>135</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Mouse Navigation</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>rulerModeTab</tabstop>
  <tabstop>lineMetricsCombo</tabstop>
  <tabstop>pathMetricsCombo</tabstop>
  <tabstop>mouseCheck</tabstop>
  <tabstop>deleteButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
